srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -53,6 +220,18 @@ public RiskModel getRiskModel()
     {
         return fRiskModel;
     }
+
+    /**
+     * 
+     * Note: this will take the place of most of the setter methods for individual fields.
+     * @param impModel
+     */
+//    public void setImportedModel( RiskModel impModel ) 
+//    {
+//        /// ??? should this also overwrite the possibly edited display name?        
+//        importedModel = impModel;
+//        importedModel.setDisplayName( modelName );
+//    }
```

Comment:
I can uncomment it even though it currently isn't being called.
Basically the importedModel is a place holder for the follow up task of importing a Model. I'm assuming that a RiskModel object will be used to hold the imported model but this is up to whoever does the task. I'm sure this code will be changed soon.